src/modules/database.ts

```diff
@@ -220,7 +220,7 @@ export class DatabaseAPI {
                 if (logErrors) log(`Error loading account${accounts.length > 1 ? 's' : ''}: ${names.join(', ')}`, logErrors ? LogLevel.Warning : LogLevel.Debug);
                 return [];
             }
-            const missing = names.filter((name) => accounts.filter((r) => r.name === name).length === 0);
+            const missing = names.filter((name) => !accounts.some((r) => r.name === name));
```

Comment:
This could use a set, instead iterating linearly over accounts many (many?) times.

Reply:
Added it